bespin/backend.py

```diff
@@ -0,0 +1,13 @@
+from gcb_web_auth.backends.oauth import OAuth2Backend
+
+BESPIN_USER_GROUP = 'duke:group-manager:roles:bespin-users'
```

Comment:
Let's move this to settings.py so it doesn't require different backends for dev/prod/etc.